Orange/widgets/data/owselectrows.py

```diff
@@ -144,8 +144,8 @@ def add_row(self, attr=None, condition_type=None, condition_value=None):
         attr_combo.setCurrentIndex(attr or 0)
         self.cond_list.setCellWidget(row, 0, attr_combo)
 
-        index = QtCore.QPersistentModelIndex(model.index(row, 3))
-        temp_button = QtGui.QPushButton('Remove', self)
+        index = QPersistentModelIndex(model.index(row, 3))
+        temp_button = QPushButton('X', self, flat = True)
```

Comment:
Would you consider using Cartesian product sign instead: `×`? It's square-shaped and valigned.